internal/config/audit.go

```diff
@@ -14,6 +14,11 @@ type AuditConfig struct {
 	Buffer BufferConfig `json:"buffer,omitempty" mapstructure:"buffer"`
 }
 
+// Enabled returns true if any nested sink is enabled
+func (c *AuditConfig) Enabled() bool {
+	return c.Sinks.LogFile.Enabled
+}
```

Comment:
Yeah was thinking this would probably be better, because we probably need to refer to this value more often than not.

Reply:
Totally, we now have a single place to update where audit is deemed enabled.